### PR TITLE
Delete obsolete vim footer in files

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by okurz** [comment]
Could the project owners please merge and add me as maintainers?

File: tests/boot.pm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2014 SUSE Linux GmbH
+# Copyright (C) 2014-2017 SUSE LLC
```
Comment (foursixnine) [inline tests/boot.pm]: You mean 2018 right?

Comment (okurz) [reply tests/boot.pm]: No, because I did not change the files in 2018, the commits were done more than a year ago

**Comment by coolo** [comment]
added tests-maintainers

**Comment by okurz** [comment]
thx

**Pull request merged**